discretize/TreeMesh.py

```diff
@@ -162,27 +170,37 @@ def __dirty__(self, val):
         self.__dirtySets__ = True
 
         deleteThese = [
-            '__sortedCells',
-            '_gridCC',
-            '_gridN', '_gridFx', '_gridFy', '_gridFz',
-            '_gridEx', '_gridEy', '_gridEz',
-            '_area', '_edge', '_vol',
-            '_faceDiv', '_edgeCurl', '_nodalGrad',
-            '_aveFx2CC', '_aveFy2CC', '_aveFz2CC', '_aveF2CC', '_aveF2CCV',
-            '_aveEx2CC', '_aveEy2CC', '_aveEz2CC', '_aveE2CC', '_aveE2CCV',
-            '_aveN2CC',
-        ]
+                        '__sortedCells',
```

Comment:
fyi the above deleted spacing also works for pep8. 

@rowanc1: thoughts on making a few notes for a style guide for code? We don't need to be overly strict on things, but might be worth pointing out preferred conventions.